gatsby-config.js

```diff
@@ -167,13 +167,6 @@ module.exports = {
         path: `${__dirname}/src/markdown-pages`,
       },
     },
-    {
-      resolve: 'gatsby-source-filesystem',
-      options: {
-        name: 'quickstarts',
-        path: `${__dirname}/src/data/quickstarts.json`,
-      },
-    },
     {
       resolve: 'gatsby-transformer-json',
```

Comment:
One last thing to remove! This transformer is only used for the quickstart json objects